src/component/HoverRepo/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './style.css';
+
+class HoverRepo extends React.Component {
```

Comment:
I dont know if it's a known bug but the hover functionality is reacting like onClick not onHover, i'll check the code it's being imported to on this though. When I try to use it, if I click, the language/stargazers/watcher pop up appears for a few seconds and then the page responds to the hyperlink and takes me to the associated github repo